FirebaseCombineSwift/Sources/Database/DatabaseReference+Combine.swift

```diff
@@ -0,0 +1,185 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#if canImport(Combine) && swift(>=5.0) && canImport(FirebaseDatabase)
+
+  import Combine
+  import FirebaseDatabase
+  import FirebaseDatabaseSwift
+
+  @available(swift 5.0)
+  @available(macOS 10.15, iOS 13, watchOS 6, tvOS 13, *)
+  public extension DatabaseReference {
+    // MARK: - Setting values on a DatabaseReference
+
+    /**
```

Comment:
For new Swift code, we've been using triple-slash comments, see https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseCombineSwift/Sources/Firestore/CollectionReference%2BCombine.swift#L31, for example.

Can you update all comments in this PR accordingly?